src/PKSim.Infrastructure/InfrastructureRegister.cs

```diff
@@ -58,9 +57,18 @@ public static void Initialize()
 
          registerRunOptionsIn(container);
 
+         registerLogging(container);
+
          EnvironmentHelper.ApplicationName = () => "pksim";
       }
 
+      private static void registerLogging(IContainer container)
+      {
+         var loggerFactory = new LoggerFactory();
```

Comment:
This is the one and single instance of the LoggerFactory used to create logger 
By default we could always log to debug window